README.md

```diff
@@ -56,7 +56,7 @@ In the above example, the Logstash pipeline defines a file input that reads from
 
 - Path to SSL bundle file.
 
-`config :ssl_ca_bundle_path, :validate => :string, :default =>  "/etc/ssl/certs/ca-bundle.crt"`
+`config :ssl_ca_bundle_path, :validate => :string, :default => nil`
```

Comment:
Ok, I'm going to suggest a slightly different approach here.  What if we leave this config option as is.  Instead, we add in an "append Scalyr cert" (you can come up with a better name) option.  If that is true (and it defaults to true), then we copy what is in `ssl_ca_bundle_path` and add in the embedded Scalyr cert.

That way we can rely both on the system certs and our embedded certificate.  This might be necessary if they are proxying our traffic through an SSL proxy on their end... And just also future proofs us a bit.  The Scalyr CA cert expires in 4 years.

Reply:
I thought the whole issue is that SSL doesn't read the bundle correctly and fails even if it has one sufficient cert, if we append ours to the non-working bundle that shouldn't fix this I would think.

Reply:
Well I guess I misunderstood something because it does seem to work.